source/configuration_manual/mail_location/obox/dictmap.rst

```diff
@@ -44,6 +44,54 @@ Cassandra/sproxyd Example Configuration
      fts_dovecot_fs = fts-cache:fscache:512M:/var/cache/fts/%4Nu:compress:zstd:3:dictmap:proxy:dict-async:cassandra ; sproxyd:http://sproxyd.scality.example.com/?class=1&reason_header_max_length=200 ; dict-prefix=%u/fts/
    }
 
+Cassandra/azure Example Configuration
```

Comment:
I would suggest restructuring the headings:

h1. fs-dictmap Configuration

h2. Example Configurations

h3. Cassandra/sproxyd
h3. Cassandra/azure
h3. Cassandra/aws-s3

h2. Dictmap Parameters
h3. Cassandra/s3

Reply:
Okay i added some more structure. How do you like it now?